controller/client/client.go

```diff
@@ -268,6 +268,50 @@ func (c *Client) GetApp(appID string) (*ct.App, error) {
 	return app, c.Get(fmt.Sprintf("/apps/%s", appID), app)
 }
 
+// GetDeployment returns a deployment queued on the deployer.
+func (c *Client) GetDeployment(deploymentID string) (*ct.Deployment, error) {
+	res := &ct.Deployment{}
+	return res, c.Get(fmt.Sprintf("/deployments/%s", deploymentID), res)
+}
+
+func (c *Client) CreateDeployment(appID, releaseID string) (*ct.Deployment, error) {
+	deployment := &ct.Deployment{}
+	return deployment, c.Post(fmt.Sprintf("/apps/%s/deploy", appID), &ct.Release{ID: releaseID}, deployment)
+}
+
+func (c *Client) StreamDeployment(deploymentID string, output chan<- *ct.DeploymentEvent) (stream.Stream, error) {
+	return c.Stream("GET", fmt.Sprintf("/deployments/%s", deploymentID), nil, output)
+}
+
+func (c *Client) DeployApp(appID, releaseID string) (initial bool, err error) {
+	d, err := c.CreateDeployment(appID, releaseID)
+	if err != nil {
+		return true, err
+	}
+
+	// if initial deploy, just stop here
+	if d.ID == "" {
+		return true, nil
```

Comment:
Yeah, we could use it in [the receiver](https://github.com/flynn/flynn/blob/deploy-strategy/receiver/flynn-receive.go#L136), or we can remove that parameter, which would make it neater to use, since we can just `return client.DeployApp`